includes/islandora_scan_batch.inc

```diff
@@ -379,7 +379,7 @@ class IslandoraScanBatchObject extends IslandoraBatchObject {
             $mods_datastream->setContentFromFile($this->objectInfo['xml']->uri, FALSE);
           }
           // MARCXML, transform to MODS and set.
-          elseif ($s_xml->getName() == 'record') {
+          elseif ($s_xml->getName() == 'collection' || 'record') {
```

Comment:
Would always be `TRUE`: Would be evaluated as `($s_xml->getName() == 'collection') || 'record'`, and a non-empty string is `TRUE`.

Something like: `elseif (in_array($s_xml->getName(), array('collection', 'record')) {` should do the trick... I think?